AppCenterDistribute/AppCenterDistributeTests/MSDistributeAppDelegateTests.m

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#import <AppCenterDistribute/AppCenterDistribute.h>
```

Comment:
Why is this using umbrella header?

Reply:
"Definition of 'MSDistribute' must be imported from module 'AppCenterDistribute.MSDistribute' before it is required"

Reply:
```suggestion
#import "MSDistribute.h"
```